apps/src/p5lab/spritelab/constants.js

```diff
@@ -0,0 +1,200 @@
+/** @file Sprite Lab constants */
+var utils = require('@cdo/apps/utils');
+
+module.exports.LocationPickerMode = utils.makeEnum('IDLE', 'SELECTING');
+
+module.exports.CostumeCategories = {
+  category_animals: 'Animals',
+  category_generic_items: 'Generic items',
+  category_vehicles: 'Vehicles',
+  category_characters: 'Characters',
+  category_environment: 'Environment',
+  category_food: 'Food',
+  category_tools: 'Tools',
+  category_gameplay: 'Board games',
+  category_obstacles: 'Obstacles',
+  category_all: 'All'
+};
+
+module.exports.SpritelabReservedWords = [
+  // Gamelab globals
```

Comment:
NIT: Spritelab globals

Reply:
Good catch. Went through and changed a bunch of these reserved words since the interpreted library is way smaller than it used to be